test/browser/logging-methods.js

```diff
@@ -146,6 +146,25 @@
         sandbox.restore()
       })
 
+      it('adds padding (if specified) to short logger names', function () {
+        try {
+          var abc = new Logdown({prefix: 'abc'})
+          var demo = new Logdown({prefix: 'demo', alignOuput: true})
+          var longDemo = new Logdown({prefix: 'longDemo', alignOuput: true})
+          var longerDemo = new Logdown({prefix: 'longerDemo', alignOuput: true})
+
+          assert(abc.prefix.length === 3)
+          assert(demo.prefix.length === 10)
+          assert(longDemo.prefix.length === 10)
+          assert(longerDemo.prefix.length === 10)
+        } catch (error) {
+          sandbox.restore()
+          throw error
+        }
+
+        sandbox.restore()
+      })
+
```

Comment:
I really appreciate that logdown.js works on Node.js and in modern browsers (that was a key feature for me!). But I am a little surprised that there are the same tests in in `browser/logging-methods.js` and `server/logging-methods.js`. We should not have code duplication across those files. But to fulfill the request for now, I duplicated the test case.